src/catkin_pkg/package_templates.py

```diff
@@ -41,6 +41,8 @@
 from catkin_pkg.package import Package
 from catkin_pkg.package import PACKAGE_MANIFEST_FILENAME
 from catkin_pkg.package import Person
+from catkin_pkg.cmake import get_metapackage_cmake_template_path
+from catkin_pkg.cmake import configure_file
```

Comment:
Sorry for nitpicking: but following PEP8 the imports should be in alphabetical order. Please move the lines above the other `catkin_pkg` imports and flip their order, first `configure_file`, then `get_metapackage_cmake_template_path`.